benchmarks/Microsoft.AspNetCore.Routing.Performance/Matching/MatcheFindCandidateSetSingleEntryBenchmark.cs

```diff
@@ -9,9 +9,12 @@ namespace Microsoft.AspNetCore.Routing.Matching
 {
     public class MatcheFindCandidateSetSingleEntryBenchmark : MatcherBenchmarkBase
     {
+        // SegmentCount should be max-segments + 1
+        private const int SegmentCount = 2;
+
         private TrivialMatcher _baseline;
         private DfaMatcher _dfa;
-
+        
```

Comment:
nit

Reply:
huh. I really thought I told VS to do the automatic code cleanup then and fix this. I really dream of a day when I don't have to deal with this kind of thing anymore. Anyone have gofmt for c#?